uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -114,8 +127,19 @@ fun AppContent(
 }
 
 private class AppActions(navController: NavHostController) {
-    val onSelectDrawerItem: (String) -> Unit = { route ->
-        navController.navigate(route)
+    val onSelectDrawerItem: (DrawerContents) -> Unit = { contents ->
+        navController.navigate(contents.route) {
+            popUpTo(navController.graph.startDestination) {
```

Comment:
Maybe it's a difficult code for those who are new to it, so could you please add a comment? 🙇

Reply:
I added a comment. Would it be okay? :eyes: